README

```diff
@@ -95,6 +116,28 @@ By default `make install` will install into `/usr/local`, for changing the path
     dependencies: *pkg-config*(build-time), *libnl-3* and *libnl-genl-3*
     default: *check*
 
+## Runtime dependencies:
+`htop` has a set of fixed minimum runtime dependencies, which is kept as minimal as possible:
+* `ncurses` libraries for terminal handling (wide character support).
+
+### Runtime optional dependencies:
+`htop` has a set of fixed optional dependencies, depending on build/configure option used:
+* `libdl`, if not building static and support for some of the optional libraries is enabled, is always required when support for to optionally load dependencies (i.e. `libsensors`, `systemd`) is present.
+* `libcap`, user-space interfaces to the POSIX 1003.1e, is always required when `--enable-capabilities` was used to configure `htop`.
+* `libsensors`, readout of temperatures and CPU speeds, is optional even when `--enable-sensors` was used to configure `htop`.
+* `systemd` is optional when `--enable-static` was not used to configure `htop` (Linux only). If building statically and `libsystemd` is not found by `configure` support for the SystemD meter is disabled entirely.
```

Comment:
Please check the grammar in this line.

Reply:
I have looked at it, the line is a copy from [you](https://github.com/htop-dev/htop/pull/639#discussion_r650382302),

> * `systemd` is optional when `--enable-static` was not used to configure `htop` (Linux only). If building statically and `libsystemd` is not found by `configure` support for the SystemD meter is disabled entirely.

It looks good to me.

Reply:
k. Got tripped by the support after `configure` being part of the action, not the condition in that sentence. So while grammatically correct it's probably not the best way to phrase it as it you can get trip easily while trying to parse that sentence.